BUILD.bazel

```diff
@@ -209,7 +209,6 @@ GOLDEN_UPDATING_UNIT_TESTS = [
     args = [test_class],
     runtime_deps = [
         ":gapic_generator_java",
-        ":gapic_generator_java-test",
```

Comment:
why this change?

Reply:
`gapic_generator_java-test` does not exist, `bazel test //...` would error out without this change.